src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -568,9 +568,7 @@ public void GetAsync_ServerNeedsAuthAndNoCredential_StatusCodeUnauthorized()
             }
         }
 
-        [ActiveIssue(23769)]
         [ActiveIssue(22707, TestPlatforms.AnyUnix)]
-        [OuterLoop] // TODO: Issue #11345
```

Comment:
Do not remove the outerloop attribute.  Currently, all networking tests are outerloop especially those that connect with external servers.

Also, I don't understand why this is being enabled when the underlying reason for disabling it hasn't really been addressed.

Reply:
My intention is not to remove the Outloop attribute but only to use for my validations, that's why the "NO MERGE" label.

I got confused with other bug that was to check if it was still an issue, the intermittent failures described in the original bug are still happening per comment above. I will investigate those.